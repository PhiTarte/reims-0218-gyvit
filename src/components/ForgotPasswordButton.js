import React from "react"
import { Button } from "reactstrap"

const ForgotPasswordButton = () => (
  <div>
    <Button
      style={{
        width: "192px",
        height: "54px",
        "border-radius": "100px",
        "background-image": "linear-gradient(to left, #57aad1, #7a57d1)"
      }}
      color="info"
    >
      Request reset link
    </Button>
  </div>
)

export default ForgotPasswordButton
