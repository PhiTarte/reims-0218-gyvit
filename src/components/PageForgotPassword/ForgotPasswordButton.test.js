//TODO CREATE TEST

import React from "react"
import ReactDOM from "react-dom"
import renderer from "react-test-renderer"
import ForgotPasswordButton from "./ForgotPasswordButton"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<ForgotPasswordButton />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders link html", () => {
  const tree = renderer.create(<ForgotPasswordButton />).toJSON()
  expect(tree).toMatchSnapshot()
})
