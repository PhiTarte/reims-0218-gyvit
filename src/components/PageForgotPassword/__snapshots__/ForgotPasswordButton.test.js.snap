// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders link html 1`] = `
<div>
  <button
    className="btn btn-info"
    onClick={[Function]}
    style={
      Object {
        "backgroundImage": "linear-gradient(to left, #57aad1, #7a57d1)",
        "borderRadius": "100px",
        "height": "54px",
        "width": "192px",
      }
    }
    type={undefined}
  >
    Request reset link
  </button>
</div>
`;
